L/libpolymake_julia/common.jl

```diff
@@ -49,12 +50,14 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("CompilerSupportLibraries_jll"),
-    BuildDependency(PackageSpec(name="Julia_jll", version=v"1.4.1")),
+    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll"),
-    Dependency(PackageSpec(name="polymake_jll", version=v"4.2.1")),
+    Dependency(PackageSpec(name="polymake_jll", version=v"400.200.100")),
     BuildDependency(PackageSpec(name="GMP_jll", version=v"6.1.2")),
     BuildDependency(PackageSpec(name="MPFR_jll", version=v"4.0.2")),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+    preferred_gcc_version=v"8",
```

Comment:
We need GCC 8 or newer to make sure to avoid that libcxxwrap bug (but @benlorenz discovered this, so he's aware -- I just wanted to call this change out, so it doesn't slip by).